python_hll/serialization.py

```diff
@@ -255,7 +255,7 @@ def write_word(self, word):
             aligned_bits = BitUtil.left_shift_long(bits_that_the_byte_can_accept, bits_in_byte_remaining_after_write)
 
             # Update the byte with the alignedBits.
-            self._bytes[self._byte_index] |= BitUtil.to_signed_byte(aligned_bits)
+            self._bytes[self._byte_index] |= aligned_bits
```

Comment:
The corresponding Java code explicitly converts the long to a byte, which is why we do the same here:

https://github.com/aggregateknowledge/java-hll/blob/1f4126e79a85b79581460c75bf1c1634b8a7402d/src/main/java/net/agkn/hll/serialization/BigEndianAscendingWordSerializer.java#L151-L152